be/src/storage/task/engine_manual_compaction_task.cpp

```diff
@@ -55,7 +34,8 @@ namespace starrocks {
 EngineManualCompactionTask::EngineManualCompactionTask(MemTracker* mem_tracker, TTabletId tablet_id,
                                                        bool base_compaction)
         : _tablet_id(tablet_id), _base_compaction(base_compaction) {
-    _mem_tracker = mem_tracker;
+    _mem_tracker =
+            std::make_unique<MemTracker>(-1, "manual_compact_task_of_tablet_" + std::to_string(tablet_id), mem_tracker);
```

Comment:
better to use `fmt::fomrat` rather than `"manual_compact_task_of_tablet_" + std::to_string(tablet_id)`

Reply:
Fixed